packages/go_router/lib/src/route_data.dart

```diff
@@ -233,6 +236,12 @@ class TypedGoRoute<T extends GoRouteData> extends TypedRoute<T> {
   ///
   final String path;
 
+  /// The name that corresponds to this route.
+  ///
+  /// See [GoRoute.name].
```

Comment:
Maybe mentions this is useful for analytics that reads the RouteSettings.name.

Reply:
Done.